hash/saturn.xml

```diff
@@ -18824,7 +18824,7 @@ Olympic Soccer (Fra) T-7904H-09
 	<!-- Identifying Lupin the 3rd Chronicles (Lupin + Fujiko Version) Disc 1 (T-18805G)... -->
 	<!-- Identifying Lupin the 3rd Chronicles (Lupin + Fujiko Version) Disc 2 (T-18805G)... -->
 	<software name="lupincr" supported="no">
-		<description>Lupin the 3rd Chronicles (Lupin + Fujiko Version) (Jpn)</description>
+		<description>Lupin the 3rd Chronicles (Lupin + Fujiko version) (Jpn)</description>
```

Comment:
I believe the “Version” in this case is part of the title, and should be capitalised as such.

Reply:
corrected